.github/workflows/global-replicator.yml

```
@@ -179,4 +179,22 @@ jobs:
           committer_username: asyncapi-bot
           committer_email: info@asyncapi.io
           commit_message: "ci: update of files from global .github repo"
+          bot_branch_name: bot/update-files-from-global-repo 
+    
+  replicate_docs_workflow:
+    if: startsWith(github.repository, 'asyncapi/')
+    name: Replicate update-docs workflow to repositories
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Replicating file
+        uses: derberg/manage-files-in-multiple-repositories@beecbe897cf5ed7f3de5a791a3f2d70102fe7c25
+        with: 
+          github_token: ${{ secrets.GH_TOKEN }}
+          patterns_to_include: .github/workflows/update-docs-on-docs-commit.yml
```

```suggestion
          patterns_to_include: .github/workflows/update-docs-on-docs-commits.yml
```

we need exactly the same name as people now have in their repos added manually, so it will be replaced after we merge this PR

please update the other workflow file accordingly